ui/index.ts

```diff
@@ -1,8 +1,19 @@
 import App from "./App.svelte";
 
+const parseBackendAddress = (): string | undefined => {
+  const search = window.location.search;
+
+  if (search.includes("?backend=")) {
+    const match = search.match(/\?backend=(.*)/);
+    if (match) {
+      return match[1];
+    }
+  }
+};
+
 const app = new App({
   target: document.body,
-  props: {},
+  props: { backendAddress: parseBackendAddress() },
```

Comment:
Is there an advantage to passing this down via `App` as opposed to calling `backendAddresssStore.set()` here? The latter would involve fewer components.

Reply:
Good call! I wasn't sure whether the store would work before the `App` component is bound to the DOM. But it does, and the result is much cleaner.

https://github.com/radicle-dev/radicle-upstream/pull/1492/commits/fa20392eb28f7b53d4ef4712a8c6350b61c0b42a

Reply:
Even cleaner with `qs`: https://github.com/radicle-dev/radicle-upstream/pull/1492/commits/4420975ab7cfc41d091792d380503bc29ab62e4b.